ingestion/functions/parsing/brazil_distrito_federal/distrito_federal.py

```diff
@@ -0,0 +1,196 @@
+        commorbidities_list.append(_COMORBIDITIES_MAP["Obesidade"])
+
+    preexistingConditions["values"] = commorbidities_list
+    return preexistingConditions
+
+
+def convert_demographics(gender: str, age: str):
+    if not any((gender, age)):
+        return None
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        if age == ">= 60 anos":
+            demo["ageRange"] = {"start": 60, "end": 120}
+        # No age resolution provided below 19 years
+        elif age == "<= 19 anos":
+            demo["ageRange"] = {"start": 0, "end": 19}
+        else:
+            age_range = age.split(" a ")
+            demo["ageRange"] = {"start": float(age_range[0]), "end": float(
+                "".join([i for i in age_range[1] if not i.isalpha()])), }
+    return demo
+
+
+def convert_notes(hematologic: str, immunosuppressed: str, other: str):
+    raw_notes = []
+    if immunosuppressed == "Sim":
+        raw_notes.append("Patient with immunosuppression")
+    if hematologic == "Sim":
+        raw_notes.append("Hematologic disease")
+    if other == "Sim":
+        raw_notes.append("Unspecified pre-existing condition")
+
+    notes = (", ").join(raw_notes)
+    if notes == "":
+        return None
+    return notes
+
+
+def parse_cases(raw_data_file: str, source_id: str, source_url: str):
+    """
+    Parses G.h-format case data from raw API data.
+    """
+    with open(raw_data_file, "r") as f:
+        reader = csv.DictReader(f, delimiter=";")
+        for row in reader:
+            # There are a few entries for other states
+            if row[_STATE] == "DISTRITO FEDERAL":
+                try:
+                    case = {
+                        "caseReference": {"sourceId": source_id, "sourceUrl": source_url},
+                        "location": {
+                            "query": ", ".join(
+                                [row[_MUNICIPALITY], "Distrito Federal", "Brazil"]
+                            )
+                        },
+                        "events": convert_events(
+                            row[_DATE_CONFIRMED],
+                            row[_DATE_SYMPTOMS],
+                            row[_DEATH]
+                        ),
+                        "demographics": convert_demographics(
+                            row[_GENDER], row[_AGE]
+                        ),
+                    }
+                    case["preexistingConditions"] = convert_preexisting_conditions(
+                        row[_LUNG], row[_KIDNEY], row[_METABOLIC], row[_CARDIOVASCULAR], row[_OBESITY]
+                    )
+                    notes = convert_notes(
+                        row[_HEMATOLOGIC], row[_IMMUNOSUPPRESSED], row[_OTHERS]
+                    )
+                    if notes:
+                        case["notes"] = notes
+                    yield case
+                except ValueError as ve:
+                    raise ValueError("Unhandled data: {}".format(ve))
```

Comment:
we preferred f strings instad of format() in the code, please use it for consistency:

raise ValueError(f"error converting case: {ve}")